lib/JsonpMainTemplatePlugin.js

```diff
@@ -36,7 +36,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 			"var script = document.createElement('script');",
 			"script.type = 'text/javascript';",
 			"script.charset = 'utf-8';",
-			"script.async = true;",
+			"script.async = false;",
```

Comment:
[here](https://github.com/webpack/webpack/pull/3400/commits/47290fb2786d02507a812e518c40baf513cea50d) is a change, but it probably break the tests. need to change some tests, don't know which one need to be fixed and how =(